src/main/java/seedu/duke/Duke.java

```diff
@@ -59,21 +59,21 @@ public void startService() throws DukeException {
             case MARK:
                 int markIndex = this.ui.mark();
                 if (this.taskList.isOutOfRange(markIndex)) {
-                    throw new DukeException(Ui.i5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
+                    throw new DukeException(Ui.I5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
                 }
                 this.taskList.mark(markIndex);
                 break;
             case UNMARK:
                 int unmarkIndex = this.ui.unmark();
                 if (taskList.isOutOfRange(unmarkIndex)) {
-                    throw new DukeException(Ui.i5 + "☹ OOPS!!! Please specify a valid task number to be unmarked.");
+                    throw new DukeException(Ui.I5 + "☹ OOPS!!! Please specify a valid task number to be unmarked.");
```

Comment:
```suggestion
                    throw new DukeException(Ui.I5
                        + "☹ OOPS!!! Please specify a valid task number to be unmarked.");
```
Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.